app/.storybook/i18next.js

```diff
@@ -1,14 +1,15 @@
+import Backend from "i18next-http-backend";
+import LanguageDetector from "i18next-browser-languagedetector";
```

Comment:
ℹ️  I think Prettier reordered this? Not entirely sure why it wasn't already sorted like this.